ToDoList/app/src/main/java/com/example/jong/todolist/MainActivity.java

```diff
@@ -0,0 +1,134 @@
+package com.example.jong.todolist;
+
+
+import android.app.AlertDialog;
+import android.content.ContentValues;
+import android.content.DialogInterface;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.EditText;
+import android.widget.ListAdapter;
+import android.widget.ListView;
+import android.widget.SimpleAdapter;
+import android.widget.SimpleCursorAdapter;
+import com.example.jong.todolist.R;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+
+public class MainActivity extends AppCompatActivity {
+
+    ArrayList<String> arraylist = new ArrayList();
+    ListView listView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        listView = (ListView) findViewById(R.id.todolist1);
+        listView.setAdapter(new ArrayAdapter<String>(this, R.layout.todolist_text, arraylist));
+        listView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+            /* short click to edit*/
+            @Override
+            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+                final int pos=position;
```

Comment:
You don't have to reassign position to be a final variable. You can just put the keyword final in the method signature,
so it would look like this.
public void onItemClick(AdapterView<?> parent, View view,final int position, long id)